_includes/fragments/zbt-1-2-migrate-from-backup-procedure.md

```diff
@@ -0,0 +1,158 @@
+{% steps %}
+{% step "Creating a Zigbee2MQTT backup" %}
+{% image "/static/img/connect-zbt-1/z2m-backup-01.png" "Creating a Zigbee2MQTT backup" %}
+{% stepContent %}
+
+1. Under **Settings** > **Add-ons**, open the **Zigbee2MQTT** add-on and select **Open Web UI**.
+2. In the Zigbee2MQTT UI, go to **Settings** > **Tools** and select **Request z2m backup**.
+   - Select **Download Zigbee2MQTT backup**.
+   - {% stepResult "A zip file should now be downloaded into your **Downloads** folder." %}
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Unzipping the backup folder" %}
+{% image "/static/img/connect-zbt-1/z2m-backup-restore-01.png" "Unzipping the backup folder" %}
+{% stepContent %}
+
+- In your **Downloads** folder, unzip the **z2m-backup** file.
+  - **Info**: It should contain a **coordinator_backup.json** file.
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Stopping the Zigbee2MQTT add-on" %}
+{% image "/static/img/connect-zbt-1/z2m-delete-add-on.png" "Stopping the Zigbee2MQTT add-on" %}
+{% stepContent %}
+
+- Under **Settings** > **Add-ons**, in the **Zigbee2MQTT** add-on, select **Stop**.
+- Zigbee2MQTT uses the MQTT discovery feature to make the Zigbee devices available in Home Assistant. After removing the Zigbee2MQTT add-on, the devices are still present in Home Assistant since the MQTT broker is still holding on to the MQTT discovery messages.
+- It is recommended to remove these stale devices from the MQTT broker (see next steps).
+
+{% endstepContent %}
+{% endstep %}
+
+{% step "Removing Zigbee2MQTT devices (option 1)" %}
+{% image "/static/img/connect-zbt-1/z2m-mqq-integration-new-01.png" "Removing Zigbee2MQTT devices (option 1)" %}
+{% stepContent %}
+
+If you don't use MQTT outside of Zigbee2MQTT, you can delete the **MQTT** integration.
+
+1. Under [**Settings** > **Devices & services**](https://my.home-assistant.io/redirect/integrations/), select  **MQTT** integration.
```

Comment:
nit: double space and "the" is possibly missing:

Add the and remove double space:
```suggestion
1. Under [**Settings** > **Devices & services**](https://my.home-assistant.io/redirect/integrations/), select the **MQTT** integration.
```

Only remove double space:
```suggestion
1. Under [**Settings** > **Devices & services**](https://my.home-assistant.io/redirect/integrations/), select **MQTT** integration.
```